tests/Doctrine/Tests/Models/DDC964/DDC964User.php

```diff
@@ -97,6 +112,19 @@ public function getGroups(): ArrayCollection
         return $this->groups;
     }
 
+    public function addOrganization(DDC964Organization $organization): void
+    {
+        $this->organizations->add($organization);
+    }
+
+    /**
+     * @psalm-return Collection<int, DDC964Organization>
+     */
+    public function getOrganizations(): ArrayCollection
```

Comment:
I think it could be a `PersistentCollection`

```suggestion
    public function getOrganizations(): Collection
```

Reply:
Done it for `getGroups` and for `DDC964Group` too.